packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,245 @@
+# Bottom Sheet
+
+Bottom sheet is a container that presents additional information to a user in an overlay.
+
+This document outlines the API of `BottomSheet` component.
+
+<img src="./bottomsheet-thumbnail.png" width="380" alt="Thumbnail of bottom sheet" />
+
+- [Bottom Sheet](#bottom-sheet)
+  - [Design](#design)
+  - [Anatomy](#anatomy)
+  - [API](#api)
+    - [`BottomSheet`](#bottomsheet)
+    - [`BottomSheetBody`](#bottomsheetbody)
+    - [`BottomSheetHeader` API](#bottomsheetheader-api)
+    - [`BottomSheetFooter` API](#bottomsheetfooter-api)
+  - [Composition with Dropdown](#composition-with-dropdown)
+  - [Accessibility](#accessibility)
+  - [Open questions](#open-questions)
+  - [Alternative APIs](#alternative-apis)
+  - [References](#references)
+
+## Design
+
+- [Figma - BottomSheet](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=26477%3A578228&t=1RMqjs99P33Udj0C-0)
+
+## Anatomy
+
+<img src="./bottomsheet-anatomy.png" alt="BottomSheet Anatomy" width="100%" />
+
+
+Components:
+
+- BottomSheet
+  - BottomSheetHeader
+  - BottomSheetBody
+  - BottomSheetFooter
+
+## API
+
+Sample usage:
+
+```jsx
+import { BottomSheet } from '@razorpay/blade';
+
+<BottomSheet isOpen={boolean} snapPoints={[]}>
+  <BottomSheetHeader
+    title="Select Account"
+    leading={<BladeIcon />}
+  />
+  <BottomSheetBody>
+    <Text>Body Content<Text>
+  </BottomSheetBody>
+  <BottomSheetFooter
+    title="Footer Title"
+    leading={BladeIcon}
+    trailing={
+      <>
+        BladeIcon | IconButton | Button
+      </>
+    }
+  />
+</BottomSheet>;
+```
+
+**Design Constraints**
+
+- BottomSheetHeader:
+  - title
+  - leading:
+    - BladeIcon
+
+- BottomSheetFooter:
+  - title
+  - leading:
+    - BladeIcon
+  - trailing
+    - any jsx element
+
+- The BottomSheetFooter will behave similar to ActionListFooter.
```

Comment:
Can you elaborate on what we mean by "behave" here? Someone shouldn't need to understand ActionListFooter to understand the constraint here

Reply:
Will remove this line, seems redundant.

Reply:
So basically BottomSheetFooter has similar API surface as ActionListFooter